Hardware/cheshire.md

```diff
@@ -0,0 +1,134 @@
+---
+riscv_hardware: true
+cmake_plat: cheshire
+xcompiler_arg: -DRISCV64=1
+platform: Cheshire
+arch: RV64IMAFDC
+virtualization: "No"
+iommu: "No"
+simulation_target: false
+Status: "Unverified"
+Contrib: "UNSW"
+Maintained: "UNSW"
+cpu: Cheshire
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Cheshire
+
+Cheshire is a 6-stage RISC-V CPU. For details, refer to
+[https://github.com/pulp-platform/cheshire](https://github.com/pulp-platform/cheshire)
+
+Cheshire currently only provides seL4 support for the [Genesys 2
+board](https://reference.digilentinc.com/reference/programmable-logic/genesys-2/reference-manual)/
```

Comment:
Not sure what this means? Do you mean that if you use the seL4 port for Cheshire it will only work on the Genesys 2?

Reply:
Yes, Cheshire can be loaded onto different FPGAs or ASICs and this port doesn't support those.